src/basic-syntax/static-and-const.md

```diff
@@ -39,9 +39,11 @@ fn main() {
 ```
 
 As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and embedded code, and the variable lives through the entirety of the program execution.
+When a globally-scoped value does not have a reason to need its own address in memory, `const` is generally preferred.
 
 
 We will look at mutating static data in the [chapter on Unsafe Rust](../unsafe.md).
+Because `static` variables are accessible from any thread, mutable static variables require manual, unsafe, synchronization of accesses.
```

Comment:
This could link to https://google.github.io/comprehensive-rust/unsafe/mutable-static-variables.html which talks about mutable statics a bit more.

Reply:
Maybe it would make more sense to change the link in the previous sentence:
```
We will look at [mutating static data](../unsafe/mutable-static-variables.md) in the chapter on Unsafe Rust.
```

Reply:
Sure, go for it.